lib/fsm/door_fsm.ipp

```diff
@@ -0,0 +1,55 @@
+#ifndef PET_PASSAGE_FSM_DOOR_FSM_I_
```

Comment:
Same inconsistency as the previous file.

Reply:
see: https://github.com/Pet-Passage/Pet-Passage/pull/2#discussion_r671465045